elastalert/elastalert.py

```diff
@@ -279,7 +282,10 @@ def get_index_start(self, index, timestamp_field='@timestamp'):
         """
         query = {'sort': {timestamp_field: {'order': 'asc'}}}
         try:
-            res = self.current_es.search(index=index, size=1, body=query, _source_include=[timestamp_field], ignore_unavailable=True)
```

Comment:
Mar 19 15:30:52 search01 elastalert[22967]: ERROR:root:Error running query: RequestError(400, u'illegal_argument_exception', u'request [/filebeat-*/_search] contains unrecognized parameter: [_source_include] -> did you mean any of [_source_includes, _source_excludes]?')

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.0/docs-get.html

If you only need one or two fields from the complete _source, you can use the _source_include & _source_exclude parameters to include or filter out that parts you need.

vs

https://www.elastic.co/guide/en/elasticsearch/reference/7.0/docs-get.html

If you only need one or two fields from the complete _source, you can use the _source_includes and _source_excludes parameters to include or filter out the parts you need.